spring-cloud-sleuth-stream/src/main/java/org/springframework/cloud/sleuth/stream/SleuthStreamProperties.java

```diff
@@ -44,4 +45,40 @@ public void setEnabled(boolean enabled) {
 	public void setGroup(String group) {
 		this.group = group;
 	}
+
+	public Poller getPoller() {
+		return this.poller;
+	}
+
+	public void setPoller(Poller poller) {
+		this.poller = poller;
+	}
+
+	public static class Poller {
+		/**
+		 * Fixed delay (ms). Default: 1000
+		 */
+		private long fixedDelay = 1000L;
```

Comment:
These comments produce the comments in the Spring Boot Configuration Metadata (for content assist in the IDE). The alternative to this is creating the `additional-spring-configuration-metadata.json` file. I am impartial to either, whichever the project prefers.